libs/store/src/lib/reducers/remove.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  AppStateModel,
+  AppPlayerLogId,
+  AppRaceId,
+  AppPlayerId,
+} from '@razor/models';
+import { removePlayerReducerPayload } from '../payloads';
+
+// Basic Remove Operations
+export const removePlayerReducer = (
+  state: AppStateModel,
+  payload: removePlayerReducerPayload,
+): AppStateModel => {
+  const { tournamentId, playerId } = payload;
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { [playerId]: removedPlayer, ...newPlayerModel } = state.playersModel;
+  const newState: AppStateModel = {
+    ...state,
+    tournamentsModel: {
+      ...state.tournamentsModel,
+      [tournamentId]: {
+        ...state.tournamentsModel[tournamentId],
+        playerIds: [
+          ...state.tournamentsModel[tournamentId].playerIds.slice(
+            0,
+            state.tournamentsModel[tournamentId].playerIds.indexOf(playerId),
+          ),
+          ...state.tournamentsModel[tournamentId].playerIds.slice(
+            state.tournamentsModel[tournamentId].playerIds.indexOf(playerId) +
+              1,
+          ),
+        ],
+      },
+    },
+    playersModel: {
+      ...newPlayerModel,
+    },
+  };
+  return newState;
+};
+
+export const removeTournamentReducer = (
```

Comment:
Since your algorithm here includes multiple steps, please add a small comment before each code stub; so a person reading would easily understand what's being done

Reply:
Eg: `// get all player ids of all races in tournament`

Reply:
Completed docs in 38e16c224293b3eabf0be79a998e2ba44a2b93fb#diff-52d528bd5985ce267ff3fbb8716ae266bf30ae20b7c8ba1e32c4bce4749c966d